src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -146,15 +182,15 @@ function formReadReceiptsTooltip(
   const displayRecipientType = recipientType === "guardian" ? "parent" : recipientType;
   const recipientString = readBy.length === 1 ? displayRecipientType : `${displayRecipientType}s`;
   return (
-    <FlexBox alignItems="center" justify="end">
+    <FlexBox alignItems="center" justify="end" className={cssClass("readReceipts--container")}>
```

Comment:
There was some extra space that was hoverable on top of the read receipts tooltip so moving this class up to match the translated languages one looks correct